server/apps/auth/db/commands.py

```diff
@@ -91,6 +91,7 @@ def run(self, username, password):
         }
         service = superdesk.get_resource_service('auth')
         id = str(service.post([credentials])[0])
+        print('Session ID:', id)
```

Comment:
Actually is needed to know the session for which the token is generated, this is  a flask command.